recipes/tvm/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "tvm" %}
+{% set version = "0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dmlc/tvm/archive/v{{ version }}.tar.gz
+  sha256: a27926999fd157fd3727b93e5d706e935db91b9fdfc432d6328ca578a7903644
+  patches:
+   # Patching cmake to avoid to use some hardcoded paths
+    - cmake_patch.patch
+    - find_llvm.patch
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - compiler-rt
+    - cmake
+    - clangdev
+    - llvmdev
```

Comment:
`clangdev`, `llvmdev`, `compiler-rt` should be in `host` as they are used as libraries.